app/WebParsing/UtsgJsonParser.hs

```diff
@@ -110,4 +149,4 @@ instance FromJSON DBList where
         courses <- pageableCourse .: "courses"
         dbList <- mapM parseJSON courses
         return $ DBList dbList
-      Nothing -> return $ DBList []
+      Nothing -> return $ DBList []
```

Comment:
The GitHub diff is showing a change on this line, either you accidentally deleted a new line or added an extra one. In general, each file should end with exactly one blank line (and I recommend you configure your text editor to make this happen automatically).